src/DotNetStatus/assets/css/DataTable.css

```diff
@@ -0,0 +1,16 @@
+﻿/*  Copyright (c) Microsoft. All rights reserved.
+    Licensed under the MIT license. See LICENSE file in the project root for full license information.
+*/
+
+.tableHeader {
```

Comment:
Can you give these classes a better name? .tableHeader, etc is not super clear - maybe .status-table-header, .status-table-footer, etc (the hyphen is a convention I've seen around the CSS world instead of camelCase)